main.py

```diff
@@ -51,11 +51,18 @@ async def list_directory(directory: str = "") -> List[str]:
 
     Args:
         directory: The path of the directory to search for.
+
+    Returns:
+        A list with items in the given directory.
+        It lists directories before files, and is sorted alphabetically internally.
```

Comment:
The second clause looks a little complicated. How about "... before files, sorted in an alphabetical order."